packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -219,7 +220,7 @@ function MongoNav({
     },
     orgNav: {
       leaf: data?.currentOrganization
-        ? `${hosts.cloud}/v2#/org/${data?.currentOrganization?.orgId}/`
+        ? `${hosts.cloud}/v2#/org/${data.currentOrganization?.orgId}/`
```

Comment:
```suggestion
        ? `${hosts.cloud}/v2#/org/${data.currentOrganization.orgId}/`
```

Should also be valid then?

Reply:
👍